### PR TITLE
Refator keychain with main bundle

### PR DESCRIPTION
**Pull request by jaelim-ms**
Remove a parameter for service name and use unique identifier which is bundle identifier.

### CODE REVIEW

File: MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m [hunk 1]
```diff
@@ -3,57 +3,55 @@
...
   return nil;
 }
 
-+ (BOOL)clearForService:(NSString *)serviceName {
++ (BOOL)clear {
   NSMutableDictionary *item = [NSMutableDictionary new];
-  item[(__bridge id) kSecClass] = (__bridge id) kSecClassGenericPassword;
-  item[(__bridge id) kSecAttrService] = serviceName;
+  item[(__bridge id)kSecClass] = (__bridge id)kSecClassGenericPassword;
+  item[(__bridge id)kSecAttrService] = [[NSBundle mainBundle] bundleIdentifier];
```
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m]: We have a macro for that in Util :)

File: MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m [hunk 2]
```diff
@@ -3,57 +3,55 @@
...
 }
 
-+ (NSMutableDictionary *)generateItem:(NSString *)key service:(NSString *)serviceName {
++ (NSMutableDictionary *)generateItem:(NSString *)key {
   NSMutableDictionary *item = [NSMutableDictionary new];
-  item[(__bridge id) kSecClass] = (__bridge id) kSecClassGenericPassword;
-  item[(__bridge id) kSecAttrService] = serviceName;
-  item[(__bridge id) kSecAttrAccount] = key;
+  item[(__bridge id)kSecClass] = (__bridge id)kSecClassGenericPassword;
+  item[(__bridge id)kSecAttrService] = [[NSBundle mainBundle] bundleIdentifier];
```
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m]: We have a macro for that in Util :)

File: MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m (hunk 1 above)
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m]: Ahh that's only in the updates branch, forget about it ^^

File: MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m (hunk 2 above)
Comment (clpolet) [inline MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m]: Ahh that's only in the updates branch, forget about it ^^

File: MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m (hunk 1 above)
Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m]: Will switch that once I merge this to update branch.

File: MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m (hunk 2 above)
Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.m]: Will switch that once I merge this to update branch.

**Pull request merged**